packages/schema/src/config/vite.ts

```diff
@@ -109,5 +109,8 @@ export default defineUntypedSchema({
         },
       },
     },
+    cacheDir: {
+      $resolve: async (val, get) => val ?? (await get('cacheDir')),
```

Comment:
I don't think we have a `nuxt.options.cacheDir` option set, so this should probably be something more like:

```suggestion
      $resolve: async (val, get) => val ?? resolve(await get('rootDir'), 'node_modules/.cache/vite'),
```